cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -20,7 +20,6 @@ import (
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/rand"
```

Comment:
prefer keeping the cryptorand alias to make it clear we're not using math/rand

Reply:
I don't object to keeping the alias, but it's not actually confusing since the two packages have fairly different interfaces. e.g. https://golang.org/pkg/math/rand/ doesn't have a "Reader".

Reply:
Addressed. Keep the alias `cryptorand`. PTAL